### DISCUSSION TITLE
Upgrading firmware on BITFUNX PsxMemCard

### DISCUSSION THREAD

**Discussion by chrisp110** [dangiu/PicoMemcard#87] [Q&A]
I recently bought a [PsxMemCard](https://www.aliexpress.com/item/1005005875373266.html) and would like to modify the firmware to add new features. I understand that the design of this is based on the PicoMemcard+ so I assume the firmware is the same? Anybody know if the PCB design is based on the Raspberry Pi Pico or the RP2040-Zero version which build of the firmware is appropriate?

**Comment by whitezombie2000** [comment]
You probably won't get any help for clone cards. Come over here https://github.com/MrPalloncini/PicoMemcard . The firmware is pretty good but we still have some minor bugs. If you have a good understanding of how to program, troubleshoot issue, come and help us out, as this was made for the Bitfunx card.
